enhancements/installer/install_ca_management.md

```diff
@@ -0,0 +1,164 @@
+---
+title: install-ca-management
+authors:
+  - "@bparees"
+reviewers:
+  - "@sdodson"
+  - "@abhinavdahiya"
+  - "@adambkaplan"
+  - "@dmage"
+approvers:
+  - "@sdodson"
+  - "@crawford"
+creation-date: 2020-08-03
+last-updated: 2020-08-03
+status: provisional
+---
+
+# Install CA Management
+
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Customers have a need to provide additional CAs to their cluster.  Ideally they can do this at 
+install time so that everything works by the time the cluster comes up (if they do not, the 
+cluster may install but things like imagestreams may not import, builds may not work, or other 
+components may not function as expected).
+
+Today the installer collects an “additionalTrustBundle”, and optionally proxy host names.
+
+The additionalTrustBundle is put(not sure where exactly?) and consumed directly by some set of 
+components such as the Machine Config Operator.  This ensures that the nodes can talk to the 
+mirror registry in a disconnected install for example.
+
+In addition if a proxy host is supplied, the bundle is put into a proxy configuration (CA+hosts) 
+which many/most components consume.
+
+However when only the bundle is supplied (because no proxy is in use) the bundle consumption is 
+essentially ad hoc. This has the negative consequence that components which don’t directly consume 
+it (from where?) may not function, particularly in a disconnected install where they need a CA to 
+trust the mirror registry.  We see this problem w/ imagestream import which expects to get its CAs 
+from an image config resource which references a configmap of bundles that are explicitly 
+associated to registry hostnames.  Further complicating the situation is that the field is simply 
```

Comment:
> We see this problem w/ imagestream import which expects to get its CAs 
from an image config resource which references a configmap of bundles that are explicitly 
associated to registry hostnames.

Can you include docs/enhancement to point to details on this workflow.

Reply:
will do